apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -199,7 +201,7 @@ export default class AnimationPickerBody extends React.Component {
     if (!this.props.libraryManifest) {
       return <div>{msg.loading()}</div>;
     }
-    const {searchQuery, categoryQuery, results} = this.state;
+    let {searchQuery, categoryQuery, results} = this.state;
```

Comment:
this is probably leftover from previous changes, but this should probably stay as `const` because we shouldn't be mutating anything in `this.state`

Reply:
Great catch! This was leftover from when I had tried splitting the string at this point.